### PR TITLE
Expo - rename Fingerprint to LocalAuthentication

### PR DESCRIPTION
**Pull request by dalwadani**
With Expo V30.0.0, Fingerprint has been renamed to LocalAuthentication.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://blog.expo.io/expo-sdk-30-0-0-is-now-available-e64d8b1db2a7
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by janaagaard75 — COMMENTED** [review]
I think we should copy-paste the wording directly from docs.expo.io as much as possible. The current text is taken from that site, and they have updated it since, so the type definitions should follow along.

https://docs.expo.io/versions/v30.0.0/sdk/local-authentication

**Comment by moshfeu** [comment]
Isn't need to be part of a new version definition? Currently, our last version is 27. So we don't need to **rename** it, but create new version and update it there, right?

**Comment by janaagaard75** [comment]
That's a very good point, @moshfeu. I think the most correct solution is to support specific versions of Expo by matching the major version number here in Definitely Typed, but that would mean that we would have multiple versions in the wild at the same time, and thus that patches and so would have to be done on them all, and I don't see that happening. Let's keep the old `Fingerprint` definition and add `LocalAuthentication` beside it. And the probably add a version number in the JSDoc comment for the namespace, e.g. `Expo version <= 29` and `Expo version >= 30`. (Don't know at what version number the rename happened.)

TL;DR: I agree. :)

**Comment by moshfeu** [comment]
@janaagaard75 If I understand you correctly, I agree in case we want add a **missing** api to an **old** version. But here, we didn't create v30 so, instead of containing a **newer** api in v27, why not to wait to v30 (or wherever this api is changed) WDYT?
BTW, why @typescript-bot saies that `owner has approved this PR` when, clearly, the conversation is not finished it?

**Comment by KonstantinKai** [comment]
Hi there. In my opinion, adding new API to the old version is a bad idea. We should create new versions for new Expo API.

**Comment by dalwadani** [comment]
I think we should create a subfolder for version 27. See: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

**Review by moshfeu — APPROVED** [review]
Looks good 👍

**Review by KonstantinKai — CHANGES_REQUESTED** [review]
@dalwadani please review travis build log.

**Review by KonstantinKai — APPROVED** [review]
@dalwadani, thanks a lot

**Comment by janaagaard75** [comment]
> @janaagaard75 If I understand you correctly, I agree in case we want add a **missing** api to an **old** version. But here, we didn't create v30 so, instead of containing a **newer** api in v27, why not to wait to v30 (or wherever this api is changed) WDYT?

Didn't realize that we weren't up to date with Expo's versioning. Sorry for the confusion.
